docs/source/index.rst

```diff
@@ -143,6 +148,7 @@ contributing to the project, helping other users and discuss all things MLOps.
 
    installation
    tutorial
+   metrics
```

Comment:
this is not the correct place for this
it should be under `docs/source/guides` since this is an advanced guides